### PR TITLE
feat: add setTokenURI method to debt token

### PR DESCRIPTION
**Pull request by graemecode**
TODOs: 

- [x] Write integration test in debt_token.ts
- [x] Replace token-uri operator constant with the proper address.

### CODE REVIEW

File: test/ts/integration/debt_token.ts [hunk 1]
```diff
@@ -1170,4 +1173,47 @@ contract("Debt Token (Integration Tests)", (ACCOUNTS) => {
...
+        let tokenId: BigNumber;
+        const tokenURI = "https://www.example.com/image.jpeg";
+
+        before(async () => {
+            tokenId = debtEntries[0].getTokenId();
+        });
+
+        describe("when called by an account that has permission", () => {
+            before(async () => {
+                await debtToken.addAuthorizedTokenURIAgent.sendTransactionAsync(NON_CONTRACT_OWNER);
```
Comment (nadavhollander) [inline test/ts/integration/debt_token.ts]: Won't this fail if it doesn't come from the multisig?

Comment (graemecode) [reply test/ts/integration/debt_token.ts]: The integration tests for this contract _do not_ use the deployed DebtToken. Example:

```ts
// in integration/debt_token.ts

debtToken = new DebtTokenContract(debtTokenWeb3Contract, TX_DEFAULTS);

await debtRegistry.addAuthorizedInsertAgent.sendTransactionAsync(debtToken.address, {
            from: CONTRACT_OWNER,
});
await debtRegistry.addAuthorizedEditAgent.sendTransactionAsync(debtToken.address, {
            from: CONTRACT_OWNER,
});
```

If this is an issue we should change it in a separate PR. It was weird when I was it.

Comment (nadavhollander) [reply test/ts/integration/debt_token.ts]: Interesting -- we should _definitely_ fix that.  Can you make a task and add it to our Pivotal queue before closed beta?

File: migrations/migration_constants.js [hunk 2]
```diff
@@ -1,17 +1,28 @@
...
 const LIVE_NETWORK_ID = "live";
 const DUMMY_TOKEN_SUPPLY = 10 ** 27;
 const DUMMY_TOKEN_DECIMALS = 18;
 
+/**
+ * The secure address that is allowed to set DebtToken URIs.
+ *
+ * @type {string}
+ */
+const TOKEN_URI_OPERATOR = "0x5d497982326f641e0b374585ff7c1c1be9878560";
```
[the remark this answers is not in the document]
Comment (graemecode) [reply migrations/migration_constants.js]: This is one of Nadav's addresses. At some point we should be able to remove this.

**Pull request merged**